### PR TITLE
{MAINTANCE} Fixed authentication for root user not being applied

### PR DESCRIPTION
**Pull request by scotty-c**
This PR fixes the issue that the root user name and password was not being set properly. It was always being set as `admin` `orca` In UCP 2.0 there is a specification for the password to be 8 characters long. If you apply `orca` as the password UCP with respond with a hash password it has created. This breaks the functionality of being able to build UCP clusters at scale with this module

### CODE REVIEW

**Comment by scotty-c** [comment]
Tests are failing due to 

```
1.29s$ bundle exec rake test
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
rake aborted!
Don't know how to build task 'coverage' (see --tasks)
```

This does not fail when I run the test locally

**Comment by scotty-c** [comment]
There is another issue with the join command not working on v2.01, I am not sure if it should live in this PR or raised in another. I will continue to work on it in the PR and split it out if need be later. I have changed this to WIP until that bug is fixed in the latest release

**Comment by scotty-c** [comment]
I will open another PR to fix the join command. This is ready for review

File: Rakefile [hunk 1]
```diff
@@ -32,7 +32,7 @@ exclude_paths = [
 
 # Coverage from puppetlabs-spec-helper requires rcov which
 # doesn't work in anything since 1.8.7
-Rake::Task[:coverage].clear
+#Rake::Task[:coverage].clear
```
Comment (garethr) [inline Rakefile]: Either delete the line or leave it in, but leaving the comment isn't a good idea.

**Review by garethr — CHANGES_REQUESTED** [review]
One minor comment to resolve. I would also squash the commits into one logical commit.

File: Rakefile (hunk 1 above)
Comment (scotty-c) [reply Rakefile]: @garethr I have made the changes as you suggested

**Comment by scotty-c** [comment]
Closing to fix my fork will split this into 2 speerate PR's

**Pull request closed without merging**